calculator.py

```diff
@@ -0,0 +1,34 @@
+def multiply(a, b): 
+    return a * b
+
+def divide(a, b):
+    return a / b
+  
+def add(a, b):
+    return a + b
+
+def subtract(a, b):
+    return a - b
+ 
+def main():
+    expression = input("Введите математическое выражение: ")
+    number1, operator, number2 = expression.split()
+    number1 = float(number1)
+    number2 = float(number2)
+    result = None
+    if operator == "+":
+        result = add(number1, number2)
+    elif operator == "-":
+        result = subtract(number1, number2)
+    elif operator == "*":
+        result = multiply(number1, number2)
+    elif operator == "/":
+        result = divide(number1, number2)
+    else:
+        print("Неверный оператор")
+        return
+    print(f"Результат: {result:.2f}")
+main()
```

Comment:
```suggestion
if __name__ == "__main__":
    main()
```

Лучше использовать такую конструкцию (но это вас ещё ждёт впереди)